Doc/whatsnew/3.10.rst

```diff
@@ -306,3 +306,7 @@ Removed
 * Removed ``PyUnicode_AsUnicodeCopy()``. Please use :c:func:`PyUnicode_AsUCS4Copy` or
   :c:func:`PyUnicode_AsWideCharString`
   (Contributed by Inada Naoki in :issue:`41103`.)
+
+* Removed ``_Py_CheckRecursionLimit`` variable: it has been replaced by
+  ``ceval.recursion_critical`` of the :c:type:`PyInterpreterState` structure.
+  (Contributed by Victor Stinner in :issue:`41834`.)
```

Comment:
Shouldn't this be `ceval.recursion_limit` instead of `ceval.recursion_critical`? I checked the [`_ceval_state` struct](https://github.com/python/cpython/blob/d986d1657e1e7b50807d0633cb31d96a2d866d42/Include/internal/pycore_interp.h#L33) and could not find any indication of a `recursion_critical`. From what I can tell, recursion_critical only exists within the [`_ts (PyThreadState) struct`](https://github.com/python/cpython/blob/d986d1657e1e7b50807d0633cb31d96a2d866d42/Include/cpython/pystate.h#L59). Based on the context, I'm assuming it should be `ceval.recursion_limit`.

Also, I think the current wording of "has been replaced by" could be confused for a new member that was recently added (as someone less intimately familiar with the C-API, I had to double check ;-). So, I'd suggest it be changed to the following:

```suggestion
* Removed the ``_Py_CheckRecursionLimit`` variable. Instead, use
  ``ceval.recursion_limit`` of the :c:type:`PyInterpreterState` structure.
  (Contributed by Victor Stinner in :issue:`41834`.)
```

(The same applies to the Misc/NEWS entry)

Reply:
OoOooOOoops :-) Fixed!

Reply:
All good! :-) I had to triple check that to make sure, I thought I was missing something haha.